Formula/man-db.rb

```diff
@@ -0,0 +1,78 @@
+class ManDb < Formula
+  desc "Unix documentation system"
+  homepage "http://man-db.nongnu.org/"
+  url "https://download.savannah.gnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
+  keg_only :provided_by_macos
+
+  depends_on "pkg-config" => :build
+
+  resource "libpipeline" do
+    url "https://download.savannah.gnu.org/releases/libpipeline/libpipeline-1.5.1.tar.gz"
+    sha256 "d633706b7d845f08b42bc66ddbe845d57e726bf89298e2cee29f09577e2f902f"
+  end
+
+  def install
+    resource("libpipeline").stage do
+      system "./configure",
+        "--disable-dependency-tracking",
+        "--disable-silent-rules",
+        "--prefix=#{prefix}"
+      system "make"
+      system "make", "install"
+    end
+
+    ENV["libpipeline_CFLAGS"] = "-I#{include}"
+    ENV["libpipeline_LIBS"] = "-L#{lib} -lpipeline"
+
+    args = %W[
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+      --disable-cache-owner
+      --disable-setuid
+      --enable-static
+      --disable-shared
+    ]
+
+    # NB: Remove once man-db 2.8.6 is released
+    # https://git.savannah.gnu.org/cgit/man-db.git/commit/?id=b74c839eaa5000a18d1c396e995eca85b0e9464b
+    args += %w[
+      --without-systemdtmpfilesdir
+      --without-systemdsystemunitdir
+    ]
+
+    system "./configure", *args
+
+    # NB: Remove once man-db 2.8.6 is released
+    # https://git.savannah.gnu.org/cgit/man-db.git/commit/?id=056e8c7c012b00261133259d6438ff8303a8c36c
+    ENV.append_to_cflags "-Wl,-flat_namespace,-undefined,suppress"
+
+    system "make", "CFLAGS=#{ENV.cflags}"
+    system "make", "install"
```

Comment:
Since we're going to build a static library for `libpipeline`, rather than removing `libpipeline` afterwards, it should not be installed into the keg at the first place. Instead, changing `libpipeline`'s temporary installation prefix to `#{buildpath}/libpipeline` should work.

Reply:
Good call, @lembacon